### PR TITLE
Clarify opcode/enum ranges

### PR DESCRIPTION
**Pull request by dneto0**

### CODE REVIEW

File: include/spirv/spir-v.xml [hunk 1]
```diff
@@ -67,7 +67,24 @@
...
+         Each vendor determines the use of values in their own ranges.
+         Vendors are not required to disclose those uses.  If the use of a
+         value is included in an extension which is adopted by a Khronos
+         extension or specification, then that value's use may be permanently
+         fixed as if originally reserved in a Khronos range.
+
+         The SPIR Working Group strongly recommends:
+         - Each value is used for only one purpose.
+         - All values in a range should be used before allocating a new range.
+           For example, to avoid nused gaps in ranges, it is customary for
```
Comment (johnkslang) [inline include/spirv/spir-v.xml]: "nused" should be fixed.

Comment (dgkoch) [inline include/spirv/spir-v.xml]: "nused" typo

File: include/spirv/spir-v.xml [hunk 2]
```diff
@@ -67,7 +67,24 @@
     <!-- SECTION: SPIR-V Opcodes and Enumerants -->
     <!-- Reserve new ranges for vendors in contiguous blocks of 64
          preceding the "Future use" block below, and modify that block
-         accordingly. -->
+         accordingly.
+
+         Each vendor determines the use of values in their own ranges.
+         Vendors are not required to disclose those uses.  If the use of a
+         value is included in an extension which is adopted by a Khronos
```
Comment (johnkslang) [inline include/spirv/spir-v.xml]: "extension, which" 

or

"extension that"

**Comment by dneto0** [comment]
All feedback addressed.

**Pull request merged**